terminology_bot/bot.py

```diff
@@ -0,0 +1,404 @@
+import logging
+import hashlib
+import gettext
+from telegram import (ReplyKeyboardMarkup, ReplyKeyboardRemove)
+from telegram.ext import (Updater, CommandHandler, MessageHandler, Filters, RegexHandler,
+                          ConversationHandler)
+
+from config import get_config
+from term_collection import TermCollection
+from database import POSEnum
+
+
+t = gettext.translation('terminology_bot', '/usr/share/locale')
```

Comment:
Failed with:
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/gettext.py", line 529, in translation
    raise OSError(ENOENT, 'No translation file found for domain', domain)
FileNotFoundError: [Errno 2] No translation file found for domain: 'terminology_bot'